src/zenml/integrations/azure/orchestrators/azureml_orchestrator.py

```diff
@@ -0,0 +1,455 @@
+                        end_time=end_time,
+                        time_zone=TimeZone.UTC,
+                    )
+
+                elif schedule.interval_second:
+                    # If we are working with intervals
+                    interval = schedule.interval_second.total_seconds()
+
+                    if interval < 3600:
+                        frequency = "minute"
+                        interval = int(interval // 60)
+                    elif interval < 86400:
+                        frequency = "hour"
+                        interval = int(interval // 3600)
+                    else:
+                        frequency = "day"
+                        interval = int(interval // 86400)
+
+                    recurrence_pattern = None
+                    if frequency == "day":
+                        if schedule.start_time is not None:
+                            recurrence_pattern = RecurrencePattern(
+                                hours=schedule.start_time.hour,
+                                minutes=schedule.start_time.minute,
+                            )
+                        else:
+                            recurrence_pattern = RecurrencePattern(
+                                hours=datetime.datetime.now().hour,
+                                minutes=datetime.datetime.now().minute,
+                            )
+
+                    schedule_trigger = RecurrenceTrigger(
+                        frequency=frequency,
+                        interval=interval,
+                        schedule=recurrence_pattern,
+                        start_time=start_time,
+                        end_time=end_time,
+                        time_zone=TimeZone.UTC,
+                    )
+
+                if schedule_trigger:
+                    # Create and execute the job schedule
+                    job_schedule = JobSchedule(
+                        name=run_name,
+                        trigger=schedule_trigger,
+                        create_job=pipeline_job,
+                    )
+                    ml_client.schedules.begin_create_or_update(
+                        job_schedule
+                    ).result()
+                    logger.info(
+                        f"Scheduled pipeline '{run_name}' with recurrence "
+                        "or cron expression."
+                    )
+                else:
+                    logger.warning(
+                        "No valid scheduling configuration found for "
+                        f"pipeline '{run_name}'."
+                    )
+
+            except (HttpResponseError, ResourceExistsError) as e:
+                logger.error(
+                    "Failed to create schedule for the pipeline "
+                    f"'{run_name}': {str(e)}"
+                )
+
+        else:
+            ml_client.jobs.create_or_update(pipeline_job)
+            logger.info(f"Pipeline {run_name} has been executed.")
```

Comment:
```suggestion
            logger.info(f"Pipeline {run_name} has been started.")
```

maybe? executed sounds like it already finished somehow